python/mxnet/symbol/numpy/_symbol.py

```diff
@@ -4156,6 +4171,11 @@ def any(a, axis=None, out=None, keepdims=False):
         in which case a reference to out is returned.
     """
     return _npi.any(a, axis=axis, keepdims=keepdims, out=out)
+=======
```

Comment:
this conflict was not resolved properly.

Reply:
Sorry for that, Fixed now.
Thx